packages/react/src/themes/teams/components/Icon/svg/ProcessedIcons/icons-call-control-request.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+import { TeamsProcessedSvgIconSpec } from '../types'
+
+export default {
+  icon: ({ classes }) => (
+    <svg role="presentation" focusable="false" viewBox="8 8 16 16" className={classes.svg}>
+      <path
+        d="M12.125,20.4066315c0.2761421,0,0.5,0.2238579,0.5,0.5s-0.2238579,0.5-0.5,0.5H9.5
```

Comment:
Isn't the size of this icon too small as a default one? I was looking into the screener.io catches and it seems to be half sized compared to the `call-control-release` one

Reply:
![image](https://user-images.githubusercontent.com/1826985/59346532-c83ed700-8d12-11e9-8cef-5cdaf40b39a2.png)

Reply:
Thanks for pointing out. I had it fixed in one file, but not the other.